components/suggestion/search_suggestion_list.tsx

```diff
@@ -91,10 +121,10 @@ export default class SearchSuggestionList extends SuggestionList {
             return null;
         }
 
-        const items = [];
+        const items: JSX.Element[] = [];
         let haveDMDivider = false;
         for (let i = 0; i < this.props.items.length; i++) {
-            const item = this.props.items[i];
+            const item: Record<string, string> = this.props.items[i];
```

Comment:
same as above

Reply:
I've given the correct type for item and others.Would you please take a look.